sys-kernel/linux-firmware/files/linux-firmware-remove-rdfind-dep-and-use.patch

```diff
@@ -0,0 +1,29 @@
+--- a/copy-firmware.sh	2023-11-25 14:59:33.463158589 -0500
++++ b/copy-firmware.sh	2023-11-25 15:00:31.936152432 -0500
+@@ -69,11 +69,6 @@ if [ -z "$destdir" ]; then
+ 	exit 1
+ fi
+ 
+-if ! which rdfind 2>/dev/null >/dev/null; then
```

Comment:
Let's instead do something like:
```suggestion
-if [ -n ${LINUX_FIRMWARE_DO_DEDUPE} ] && ! which rdfind 2>/dev/null >/dev/null; then
```
and then we can avoid the need to conditionally patch => less awkward to test on bumps.

And in the ebuild:
use deduplicate && export LINUX_FIRMWARE_DO_DEDUPE=1